openpbs.spec

```diff
@@ -405,21 +405,21 @@ ldconfig %{_libdir}
 if [ "$1" != "1" ]; then
 	# This is an uninstall, not an upgrade.
 	${RPM_INSTALL_PREFIX:=%{pbs_prefix}}/libexec/pbs_preuninstall server \
-		%{version} ${RPM_INSTALL_PREFIX:=%{pbs_prefix}} %{defined have_systemd}
+		%{version} ${RPM_INSTALL_PREFIX:=%{pbs_prefix}} %{pbs_home} %{defined have_systemd}
```

Comment:
why were these changes needed?

Reply:
After dev install this file showed up in check-out list along with other files and checked it in without noticing it. We don't need this. I have removed it in the latest commit. Please check.